bundles/org.openhab.binding.kvv/src/main/java/org/openhab/binding/kvv/internal/KVVBridgeHandler.java

```diff
@@ -62,14 +62,7 @@ public KVVBridgeConfig getBridgeConfig() {
     @Override
     public void initialize() {
         updateStatus(ThingStatus.UNKNOWN);
```

Comment:
You can remove this useless line which introduces a useless thing status change.

Reply:
Done.